openmls/Cargo.toml

```diff
@@ -53,6 +53,8 @@ test-utils = [
   "dep:once_cell",
   "backtrace",
 ]
+test-skip-lifetime-validity-check = [
```

Comment:
I was thinking of using the `skip_validation.rs` Otherwise we'll get a lot of features ... 
Is there a reason not to use that skip validation approach?

Reply:
Oh, yeah good point. I forgot we had that :sweat_smile: